wercker.yml

```diff
@@ -55,7 +55,16 @@ build-release:
         name: go build
         code: |
           go build -o $GOPATH/bin/brizo
```

Comment:
With the `go build` defining an destination could this be causing the assets linking issue?